extensions/wikia/Wall/templates/Wall_newMessage.php

```diff
@@ -4,7 +4,7 @@
 			<?= AvatarService::renderAvatar($username, 50) ?>
 		</div>
 		<div class="speech-bubble-message">
-			<? if ( $wg->EnableMiniEditorExtForWall ):
+			<? if ( $showMiniEditor ):
 				echo $app->getView( 'MiniEditorController', 'Header', [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "MiniEditorController" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)